### ISSUE TITLE
Problems about running

### ISSUE THREAD

**Issue by ZoeJiang2222** [facebookresearch/generative-recommenders#313]
I have followed the instructions and when I'm running "CUDA_VISIBLE_DEVICES=0 python3 main.py --gin_config_file=configs/ml-1m/hstu-sampled-softmax-n128-large-final.gin --master_port=12345“, I get the following warning:

> /home/test/anaconda3/envs/gai/lib/python3.10/site-packages/torch/autograd/graph.py:824: UserWarning: fbgemm::dense_to_jagged: an autograd kernel was not registered to the Autograd key(s) but we are trying to backprop through it. This may lead to silently incorrect behavior. This behavior is deprecated and will be removed in a future version of PyTorch. If your operator is differentiable, please ensure you have registered an autograd kernel to the correct Autograd key (e.g. DispatchKey::Autograd, DispatchKey::CompositeImplicitAutograd). If your operator is not differentiable, or to squash this warning and use the previous behavior, please register torch::CppFunction::makeFallthrough() to DispatchKey::Autograd. (Triggered internally at /pytorch/torch/csrc/autograd/autograd_not_implemented_fallback.cpp:62.)
>   return Variable._execution_engine.run_backward(  # Calls into the C++ engine to run the backward pass
> /home/test/anaconda3/envs/gai/lib/python3.10/site-packages/torch/autograd/graph.py:824: UserWarning: fbgemm::jagged_to_padded_dense: an autograd kernel was not registered to the Autograd key(s) but we are trying to backprop through it. This may lead to silently incorrect behavior. This behavior is deprecated and will be removed in a future version of PyTorch. If your operator is differentiable, please ensure you have registered an autograd kernel to the correct Autograd key (e.g. DispatchKey::Autograd, DispatchKey::CompositeImplicitAutograd). If your operator is not differentiable, or to squash this warning and use the previous behavior, please register torch::CppFunction::makeFallthrough() to DispatchKey::Autograd. (Triggered internally at /pytorch/torch/csrc/autograd/autograd_not_implemented_fallback.cpp:62.)
>   return Variable._execution_engine.run_backward(  # Calls into the C++ engine to run the backward pass

And the training process seems to be stopped, how can I fix this problem?

**Comment by jiaqizhai** [comment]
Please see my answers to https://github.com/meta-recsys/generative-recommenders/issues/294.

As of "UserWarning: fbgemm::jagged_to_padded_dense: an autograd kernel was not registered to the Autograd key(s)"- this should be benign (at least as of a few months ago), but you should verify its behaviors independently.
